assets/javascripts/lib/discourse-markdown/bbcode-plugin.js

```diff
@@ -71,37 +71,23 @@ export function setup(helper) {
   });
 
   helper.allowList([
+    "div.bb-background",
+    "div.bb-border",
     "div.bb-center",
+    "div.bb-check",
+    "div.bb-justify",
     "div.bb-left",
+    "div.bb-ooc",
     "div.bb-right",
-    "div.bb-*",
+    "div.bb-side",
+    "div.bb-spoiler-content",
     "div[style=*]",
-    "span.bb-highlight",
-    "span[style=*]",
     "details.bb-spoiler",
-    "summary",
-    "div.bb-spoiler-content",
-    "span.bb-inline-spoiler",
-    "div.bb-background",
-    "div.bb-border",
     "span.bb-divide",
-    "div.bb-ooc",
-    "div.bb-side",
+    "span.bb-inline-spoiler",
+    "span.bb-highlight",
     "span.bb-pindent",
-    "div.bb-justify",
-    "div.bb-check",
+    "span[style=*]",
+    "summary",
   ]);
-
-  /**
-   * Allow lister for google fonts
```

Comment:
@Alteras1, I went ahead and removed this since it was commented out. Can add it back in if we needed it for other work.